adcm_client/util/stream.py

```diff
@@ -21,7 +21,7 @@ def file(path, **args):
         return list(build(repopath=path, **args).values())
     else:
         stream = io.BytesIO()
-        stream.write(io.open(path, 'rb').read())
+        stream.write(io.open(path, 'rb').read())  # pylint: disable=consider-using-with
```

Comment:
This warning should be fixed and not disabled. Asc @a-alferov for help if necessary.

Reply:
@dgusakov, that's result of some other people activity (pylint was updated in build image) so, why it should be fixed in this PR? Your change request undermines code review principles, IMHO

Reply:
If we add mutes here then we need to create a tech dept issue to fix it in the future.

Reply:
And as far as I am concerned it is a good practice to fix minor issues not directly related to the main PR scope.

Reply:
Fixed in last few commits